app/controllers/credits_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CreditsController < ApplicationController
   before_action :authenticate_user!
+  before_action :initialize_stripe
```

Comment:
Putting these in the controller works fine, but it'd probably be more DRY and reliable in the `Stripe::` object or whatever.... I'm just not sure what the most ideal situation is.

Reply:
I think this is pretty Rails-y and okay for now, but the next time we come back to Stripe code, I think extracting it into a Stripe service that can initialize itself when it is created might be a nice approach. That way, all you have to do is call on `Stripe::Service` (or whatever we call it), and it'll initialize itself whenever it's being invoked.

But again, not necessary until we actually are in this code again, IMO.